types/unit/timer/timerspec.pp

```diff
@@ -0,0 +1,4 @@
+# @summary Timer specification for systemd timers
+# @see https://www.freedesktop.org/software/systemd/man/timer.unit.html
```

Comment:
I get 404 for this. Did you mean https://www.freedesktop.org/software/systemd/man/latest/systemd.timer.html?